### ISSUE TITLE
Support for VeLO optimizer

### ISSUE THREAD

**Issue by CrazyBoyM** [PaddlePaddle/Paddle#50047]
### 需求描述 Feature Description

VeLO: Training Versatile Learned Optimizers by Scaling Up  
https://arxiv.org/abs/2211.09760  
the velo optimizer sames very good, is there any plan to support it?  

### 替代实现 Alternatives

I use this repo now:  
https://github.com/daishipeng/pytorch-optimizer  
but sames it is just a wrapper.

**Comment by CrazyBoyM** [comment]
https://github.com/google/learned_optimization

**Comment by CrazyBoyM** [comment]
I give pytorch the same issue: https://github.com/pytorch/pytorch/issues/93154  
And it has been labeled as "[module: optimizer](https://github.com/pytorch/pytorch/labels/module%3A%20optimizer) Related to torch.optim[t](https://github.com/pytorch/pytorch/labels/triaged)"

**Comment by CrazyBoyM** [comment]
Here is an article about it:  https://zhuanlan.zhihu.com/p/586277160

**Comment by pangyoki** [comment]
Thanks for your suggestion. We will feedback and evaluate this requirement about optimizer.
